pkg/utils/kubeconfig/kubeconfig.go

```diff
@@ -344,7 +347,7 @@ func getKubectlVersion() string {
 }
 
 func lockFileName(filePath string) string {
-	return filePath + ".eksctl.lock"
+	return os.TempDir() + filepath.Base(filePath) + ".eksctl.lock"
```

Comment:
This isn't quite right. This will try to place the lock file in the root (`/`) rather than the tmp dir. It might be better to do `fmt.Println(filepath.Join(os.TempDir(), fmt.Sprintf("%s.eksctl.lock", filepath.Base(filePath))))`:

https://go.dev/play/p/6bDaitdArcR

Reply:
Absolutely, a silly mistake 😞 . Thanks for the spot ✨!

Reply:
Easy mistake - I only spotted it because I've done the same. ;)